doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -116,6 +116,7 @@ Bluetooth mesh samples
 * Updated:
 
   * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * Updated samples with support for :ref:`zephyr:nrf21540dk_nrf52840`.
```

Comment:
Agree with Trond, this should be moved under Added bullet list, and remove Added in the beginning of the line:
* Support for :ref:`zephyr:nrf21540dk_nrf52840` for all samples.

Reply:
Do we need "for all samples."? The headline is "Bluetooth mesh samples"

Reply:
No need :)